x-pack/plugins/fleet/server/services/fleet_server/elastic_index.ts

```diff
@@ -47,22 +49,26 @@ export async function createAliasIfDoNotExists(
   alias: string,
   index: string
 ) {
-  const { body: exists } = await esClient.indices.existsAlias({
-    name: alias,
-  });
+  try {
```

Comment:
Just about all of these changes add `try{}` blocks and then throw a `FleetSetupError` that has the elasticsearch Error attached to the `meta` property. I added this because while coding the endpoint migration for artifacts, I was hitting elastic search errors coming out of fleet server migration, but it was impossible to determine exactly from where.